ios/FluentUI/Tooltip/Tooltip.swift

```diff
@@ -247,26 +246,8 @@ open class Tooltip: NSObject, TokenizedControlInternal {
         return anchorView.fluentTheme
     }
 
-    @objc private func themeDidChange(_ notification: Notification) {
-        guard let themeView = notification.object as? UIView, let anchorView = anchorView, anchorView.isDescendant(of: themeView) else {
-            return
-        }
-        tokenSet.update(fluentTheme)
```

Comment:
We don't need to call the new registerOnUpdate for tooltip?

Reply:
Good catch! Not sure how that got deleted.

Reply:
Ohh I removed it because Tooltip's tokenization happens a little too high. I'll need to do some light refactoring on this control to get things cleaned up. Thanks for the reminder!